### PR TITLE
Update: `ember addon ember-cli-autoprefixer`

### PR DESCRIPTION
**Pull request by alexdiliberto**
Updating the base addon configuration to adopt the newer `ember addon <name>` generated style. This allows for testing and proper conventions. I also updated your License to be the non-line break MIT format with the updated year. 

Look it over and let me know if you like it :+1:

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,32 +1,25 @@
...
+  name: 'ember-cli-autoprefixer',
+  included: function(app, parentAddon) {
+    this.app = app;
+    this.options = defaults(this.app.options.autoprefixer || {}, {
+      enabled: true
+    });
+    this.enabled = this.options.enabled;
+  },
+  postprocessTree: function(type, tree) {
+    var tree;
```
Comment (kimroen) [inline index.js]: Won't this override the `tree`-variable passed in to the function?

File: index.js [hunk 2]
```diff
@@ -1,32 +1,25 @@
...
+      tree = autoprefixer(tree, this.options);
+    }
 
-EmberCLIAutoprefixer.prototype.included = function included(app) {
-  this.app = app;
-  this.options = defaults(this.app.options.autoprefixer || {}, {
-    enabled: true
-  });
-  this.enabled = this.options.enabled;
-  delete this.options.enabled;
```
Comment (kimroen) [inline index.js]: Why did you remove this? (I'm not saying I disagree)

**Comment by kimroen** [comment]
Thank you very much for doing this - I've been planning change the format and add tests for a while. I really appreciate it!

File: index.js (hunk 1 above)
Comment (alexdiliberto) [inline index.js]: @kimroen Aghh good catch there, It was working so I presume it wasn't overriding but I should remove that var declaration

File: index.js (hunk 2 above)
Comment (alexdiliberto) [inline index.js]: I didn't understated why it was needed. If you want I can put it back in, just let me know.

**Comment by alexdiliberto** [comment]
@kimroen Yea no problem, just let me know about that `delete this.options.enabled` and I'll put in a fix for the things you commented on

File: index.js (hunk 2 above)
Comment (kimroen) [inline index.js]: I delete it because `this.options` is passed on directly as options for `broccoli-autoprefixer`, and I didn't want to pass on this extra option, just in case.

**Comment by alexdiliberto** [comment]
@kimroen OK, I have addressed the comments

https://github.com/alexdiliberto/ember-cli-autoprefixer/commit/6d3e5a7c298ba5b12476582547a07aaf03ec7415

**Comment by alexdiliberto** [comment]
@kimroen ping

any update here? Are you hesitant to merge for some reason. I can make updates if you would like.

**Comment by kimroen** [comment]
I will merge this, I just haven't had the time to look it over properly and test yet.

One thing though: I think it's better to check the actual generated files for the test instead of inspecting the elements in the running app. Something is absolutely better than nothing though :)

**Comment by kimroen** [comment]
This looks good! I'll merge it and update some stuff that has changed since this was created (like the QUnit syntax and other dependencies).

Thank you again, and sorry for the delay!
